modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/exportimport/data/handler/test/AccountGroupsAdminPortletDataHandlerTest.java

```diff
@@ -0,0 +1,160 @@
+import com.liferay.staging.StagingGroupHelper;
+
+import java.io.File;
+
+import org.hamcrest.CoreMatchers;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Balazs Breier
+ */
+@FeatureFlags(
+	featureFlags = {
+		@FeatureFlag(value = "LPD-35914"), @FeatureFlag(value = "LPD-47858")
+	}
+)
+@RunWith(Arquillian.class)
+public class AccountGroupsAdminPortletDataHandlerTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@BeforeClass
+	public static void setUpClass() {
+		FeatureFlagTestUtil.invokeFeatureFlagListeners(
+			CompanyConstants.SYSTEM, true, "LPD-35914");
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		FeatureFlagTestUtil.invokeFeatureFlagListeners(
+			CompanyConstants.SYSTEM, false, "LPD-35914");
+	}
+
+	@Test
+	public void testExportImportAccountGroups() throws Exception {
+		AccountGroup accountGroup = AccountGroupTestUtil.addAccountGroup(
+			_accountGroupLocalService, RandomTestUtil.randomString(),
+			RandomTestUtil.randomString());
+
+		Group group = _stagingGroupHelper.fetchCompanyGroup(
+			TestPropsValues.getCompanyId());
+
+		File larFile = _exportImportLocalService.exportLayoutsAsFile(
+			_exportImportConfigurationLocalService.
+				addDraftExportImportConfiguration(
+					TestPropsValues.getUserId(),
+					ExportImportConfigurationConstants.TYPE_EXPORT_LAYOUT,
+					ExportImportConfigurationSettingsMapFactoryUtil.
+						buildExportLayoutSettingsMap(
+							TestPropsValues.getUser(), group.getGroupId(),
+							false, new long[0],
+							HashMapBuilder.put(
+								PortletDataHandlerKeys.PORTLET_DATA,
+								new String[] {Boolean.TRUE.toString()}
+							).put(
+								PortletDataHandlerKeys.PORTLET_DATA + "_" +
+									AccountPortletKeys.ACCOUNT_GROUPS_ADMIN,
+								new String[] {Boolean.TRUE.toString()}
+							).build())));
+
+		_accountGroupLocalService.deleteAccountGroup(
+			accountGroup.getAccountGroupId());
+
+		_exportImportLocalService.importLayouts(
+			_exportImportConfigurationLocalService.
+				addDraftExportImportConfiguration(
+					TestPropsValues.getUserId(),
+					ExportImportConfigurationConstants.TYPE_IMPORT_LAYOUT,
+					ExportImportConfigurationSettingsMapFactoryUtil.
+						buildImportLayoutSettingsMap(
+							TestPropsValues.getUser(), group.getGroupId(),
+							false, new long[0],
+							HashMapBuilder.put(
+								PortletDataHandlerKeys.PORTLET_DATA,
+								new String[] {Boolean.TRUE.toString()}
+							).build())),
+			larFile);
+
+		Assert.assertNotNull(
+			_accountGroupLocalService.fetchAccountGroupByExternalReferenceCode(
+				accountGroup.getExternalReferenceCode(),
+				TestPropsValues.getCompanyId()));
```

Comment:
can we add here and in the AccountEntries....Test an assert on the status of the accountGroup/Account that should be "APPROVED"?